tests/test_with_credential.rs

```diff
@@ -22,14 +28,33 @@ pub async fn creds_client() -> ClientCredsSpotify {
     spotify.request_token().await.unwrap();
     spotify
 }
+#[cfg(target_arch = "wasm32")]
```

Comment:
Why not use the regular implementation? It seems like this does the same thing.

Reply:
Browser that runs wasm cannot fetch environment variables, we need to determine the value of the Credentials at compile time.
For this purpose, we have written another function using the ```env!``` macro.
Is this not a good idea?